iot-hub/Samples/device/ImportExportDevicesSample/ProcessImpExpCommands.cs

```diff
@@ -0,0 +1,438 @@
+﻿using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Azure.Storage.RetryPolicies;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ImportExportDevices
+{
+    public static class IOTHubDevices
+    {
+        //IoT Hub connection string. You can get this from the portal.
+        // Log into https://azure.portal.com, go to Resources, find your hub and select it.
+        // Then look for Shared Access Policies and select it. 
+        // Then select iothubowner and copy one of the connection strings.
+        public static string iotHubConnectionString = "<connection string to your IoT hub>";
+
+        // Connection string to the storage account used to hold the imported or exported data.
+        // Log into https://azure.portal.com, go to Resources, find your storage account and select it.
+        // Select Access Keys and copy one of the connection strings.
+        public static string storageAccountConnectionString = "<your storage account connection string>";
+
+        // Container used to hold the blob containing the list of import/export files.
+        // This is a module-wide variable.
+        public static CloudBlobContainer cloudBlobContainer;
+
+        // Name of blob container holding the work data.
+        public static string containerName = "devicefiles";
+
+        // Name of the file used for exports and imports. 
+        // This is set by the IoT SDK, and can't be changed.
+        public static string deviceListFile = "devices.txt";
+
+        // List of devices to add. This is a temporary file.
+        public static string devices_to_add = "devices_new.txt";
```

Comment:
NIT: consistent naming of camelCase instead of snake_case

Reply:
good point. Renamed it "devicesToAdd".

Reply:
For all of these please follow our (and .NET Core's) coding guidelines: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/doc/coding-style.md